### ISSUE TITLE
Filesystem warning supress-ability

### ISSUE THREAD

**Issue by roalyr** [rust-lang/rust#75997]
As described previously here https://github.com/termux/termux-packages/issues/4921#issue-565954999 this warning appears when incrementally compiling on the Android device, and there seems to be no way to disable it.

The more linked libraries are there within the workspace the more this warning pops up, consuming the terminal output.

I know I am in the minority here, but maybe, just maybe, making this warning supressable is not a big issue.

**Comment by pnkfelix** [comment]
On the one hand, this almost certainly affects a very small percentage of people.

On the other hand, it sounds extremely annoying for the people it does effect, and basically impossible to work around in the context described, apart from some compiler flag (e.g. even just exposing this message as just another lint may suffice, and *might* be very easy to implement).

@rustbot modify labels: +P-medium

**Comment by roalyr** [comment]
1.52.1 seems to have the message removed. Is this it, will it be gone?

**Comment by JMLX42** [comment]
Nope, still here in 1.70:

```
Compiling gltf_types v0.1.0 (/data/data/com.termux/files/home/aerys/lab/gltf-server/crates/gltf_types)
warning: hard linking files in the incremental compilation cache failed. copying files instead. consider moving the cache directory to a file system which supports hard linking in session dir `/data/data/com.termux/files/home/aerys/lab/gltf-server/target/debug/incremental/gltf_types-3owb8syz04ju5/s-gnzgaitxk8-b2167-working`

warning: `gltf_types` (lib) generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 9.00s
➜  gltf_types git:(1-implement-json-api-compound-documents-relationship-resolution) ✗ rustc --version
rustc 1.70.0 (90c541806 2023-05-31) (built from a source tarball)
```

**Comment by MohammedKHC0** [comment]
Any updates?
